.evergreen/run-tests.sh

```diff
@@ -191,7 +196,12 @@ if [ -z "$GREEN_FRAMEWORK" ]; then
         # causing this script to exit.
         $PYTHON -c "from bson import _cbson; from pymongo import _cmessage"
     fi
-    $PYTHON $COVERAGE_ARGS setup.py $C_EXTENSIONS test $TEST_ARGS $OUTPUT
+
+    if [ -n "$TEST_LOADBALANCER" ]; then
+        $PYTHON -m unittest discover -s test/load_balancer -v
```

Comment:
Does invoking tests like this still generate the XML output?

Reply:
Good point. Updated to use the unittest-xml-reporting command line tool.

Reply:
Behold:
<img width="1267" alt="Screen Shot 2021-05-25 at 10 31 12 AM" src="https://user-images.githubusercontent.com/5015933/119542513-7c8d6880-bd44-11eb-9734-06e3fa848fcc.png">